src/hotspot/os_cpu/windows_aarch64/os_windows_aarch64.hpp

```diff
@@ -26,6 +26,7 @@
 #define OS_CPU_WINDOWS_AARCH64_OS_WINDOWS_AARCH64_HPP
 
   static void setup_fpu();
+  static bool stub_use_cmath_impl(int intrinsicID);
```

Comment:
Something similar should be done in hotspot/os_cpu/bsd_aarch64/os_bsd_aarch64.cpp. I've pushed this patch in my GHA and the build have failed: https://github.com/AntonKozlov/jdk/runs/2447540531. Similar check can be made if you set up GHA as described in https://github.com/openjdk/jdk/pull/3510/checks